test/provision_test.py

```diff
@@ -0,0 +1,92 @@
+import json
+from test.test_util import TestUtils
+import pytest
+
+from autoprotocol.protocol import Protocol
+
+class TestProvision(object):
+    p = Protocol()
+    w1 = p.ref("w1", None, cont_type="96-pcr", discard=True) \
+        .well(0).set_volume("2:microliter")
+
+    def test_provision_well_capacity(self):
+        self.p.provision("rs17gmh5wafm5p", self.w1, "50:microliter")
+        actual_protocol_as_json = json.dumps(self.p.as_dict()['instructions'], indent=2, sort_keys=True)
+        expected_protocol_as_json = TestUtils.read_json_file('expected_provision.json')
```

Comment:
nit: let's update `protocol` -> `instruction` for subsequent references. i.e. `actual_protocol_as_json` -> `actual_instruction_as_json`, `expected_protocol_as_json` -> `expected_instruction_as_json`

Reply:
Renamed